.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,21 +2,23 @@
 
 Describe the changes or additions included in this PR.
 
-## Test Plan 
+## Test plan 
 
 How did you test the new or updated feature?
 
 ---
-If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.
 
-### Type of Change (Check all that apply)
+## Release notes
 
-- [ ] protocol change
-- [ ] user-visible impact
-- [ ] breaking change for a client SDKs
-- [ ] breaking change for FNs (FN binary must upgrade)
-- [ ] breaking change for validators or node operators (must upgrade binaries)
-- [ ] breaking change for on-chain data layout
-- [ ] necessitate either a data wipe or data migration
+Check each box that your changes affect.
 
-### Release notes
+If you check a box other than "None of the above", include information that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 
+
+- [ ] Protocol: 
+- [ ] Nodes (Validators and Full nodes): 
+- [ ] Indexer: 
+- [ ] JSON-RPC: 
+- [ ] GraphQL: 
+- [ ] CLI: 
+- [ ] Rust SDK: 
+- [ ] None of the above
```

Comment:
Do we really need to have this option? It would be less things to do if we don't list this option?

Reply:
I think the idea was for it to act as an explicit option to let the submitter know that release notes aren't required if this box is checked. I don't feel strongly about its inclusion, so will remove.

Reply:
Yeah, this is only relevant if we also have tooling that prevents lands without one of these options ticked, which is maybe a bit forceful.